src/test/kotlin/org/objectionary/ddr/TestBase.kt

```diff
@@ -74,7 +84,29 @@ interface TestBase {
     /**
      * @return name of the test being executed
      */
-    fun getTestName() = Thread.currentThread().stackTrace[4].methodName
-        .substring(5)
-        .replace(' ', '_')
+    fun getTestName(): String? {
+        Thread.currentThread().stackTrace.forEach {
+            val methodName = it.methodName
+            if (methodName.substring(0, "test ".length) == "test ") {
+                return methodName.substring("test ".length).replace(' ', '_')
+            }
+        }
+        return null
+    }
+
+    /**
+     * Deletes temporary output directory
+     *
+     * @param pathToSource path to source directory
+     * @throws IOException
+     */
+    fun deleteTempDir(pathToSource: Path) {
+        val tmpDir = File("${pathToSource}_$postfix")
+        try {
+            FileUtils.deleteDirectory(tmpDir)
+        } catch (e: IOException) {
+            logger.error(e.message)
```

Comment:
@c71n93 Maybe it would be better to re-throwing an exception like here: https://github.com/objectionary/ddr/pull/128/files